apps/examples/testcase/ta_tc/task_manager/itc/itc_taskmanager_main.c

```diff
@@ -69,32 +73,44 @@ static int not_builtin_task(int argc, char *argv[])
 void unregister_tasks(void)
 {
 	int ret;
-
-	ret = task_manager_unregister(tm_sample_handle, TM_RESPONSE_WAIT_INF);
-	TC_ASSERT_EQ("task_manager_unregister", ret, OK);
-
 	int tm_sample_handle2;
-
 	tm_appinfo_list_t *sample_list_info;
 
+	ret = task_manager_unregister(tm_sample_handle, TM_RESPONSE_WAIT_INF);
+	if (ret != OK) {
+		printf("ERROR: failed task_manager_unregister in unregister_tasks ERR: %d\n", ret);
+	}
+
 	sample_list_info = ((tm_appinfo_list_t *)task_manager_getinfo_with_name("itc_taskmanager_user1", TM_RESPONSE_WAIT_INF));
-	TC_ASSERT_NEQ("task_manager_getinfo_with_name", sample_list_info, NULL);
+	if (sample_list_info == NULL) {
+		printf("ERROR: failed task_manager_getinfo_with_name in unregister_tasks\n");
+		return;
+	}
 
 	tm_sample_handle2 = sample_list_info->task.handle;
 	task_manager_clean_infolist(&sample_list_info);
-	TC_ASSERT_NEQ("task_manager_clean_infolist", (tm_appinfo_list_t *)&sample_list_info, NULL);
+	if ((tm_appinfo_list_t *)&sample_list_info == NULL) {
```

Comment:
Please correct this check ,and do the similar things at other places.

Reply:
fixing it

Reply:
done